thomas/index.html

```diff
@@ -38,8 +38,12 @@
     <h2>Good game</h2>
     <img onclick="addFish()"  src="http://www.drodd.com/images15/fish8.png" alt="good pic" style=";">
     <p>You have <span id="fishQuantity">0</span> fish</p>
+	<h2> fish.com</h2>
+	<p>onclick="buyapond" pond           15 fish<p>
```

Comment:
The onclick attribute should be inside the p tag:
`<p onclick="buyapond">Pond: 15 fish<p>`